cli/cmd/compliance_azure_test.go

```diff
@@ -84,3 +84,84 @@ func TestSplitAzureSubscriptionsApiResponse(t *testing.T) {
 		})
 	}
 }
+
+func TestCliListAzureTenantsAndSubscriptionsWithoutData(t *testing.T) {
+	cliOutput := captureOutput(func() {
+		assert.Nil(t, cliListTenantsAndSubscriptions(new(api.AzureIntegrationsResponse)))
+	})
+	assert.Contains(t, cliOutput, "There are no Azure Tenants configured in your account.")
+
+	t.Run("test JSON output", func(t *testing.T) {
+		cli.EnableJSONOutput()
+		defer cli.EnableHumanOutput()
+		cliJSONOutput := captureOutput(func() {
+			assert.Nil(t, cliListTenantsAndSubscriptions(new(api.AzureIntegrationsResponse)))
+		})
+		expectedJSON := `{
+  "azure_subscriptions": []
+}
+`
+		assert.Equal(t, expectedJSON, cliJSONOutput)
+	})
+}
+
+/*
```

Comment:
Should these tests be commented out?

Reply:
Yes. I haven't figured out how to mock API calls in our unit test framework. I don't want to remove these tests that work once we figure that out. would that be ok? or should I remove them?

Reply:
Got it. Could we perhaps leave a TODO or some sort of note on the reasoning?

Reply:
I found a way to uncomment and run these tests, no need to add a TODO. I used the fake API sever go internal package. 💯